Docker/SDK6.0/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-focal
+ARG NUKEEPER_VERSION=0.35.0
```

Comment:
I feel like this would warrant a 0.36.0 bump if this PR gets merged. But not sure on all the interconnecting pieces.